packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2841,7 +2840,7 @@ export class ContainerRuntime
 
 		const message: BatchMessage = {
 			contents: serializedContent,
-			deserializedContent,
+			deserializedContent: JSON.parse(serializedContent), // Deep copy
```

Comment:
This is tricky for stuff involving dates and such. Did you try structuredClone or something?

Reply:
Actually, scratch that, not sure of the browser compatibility for that function